README.md

```diff
@@ -1,90 +1,240 @@
+
+The new toggle feature in the demo app allows the user to sort items by price or category, giving the user more control over their view of the products displayed on the page. To roll this feature out, it has been built behind an Optimizely-powered Experiment Variation that provides the ability to gate access without the need to deployment different versions of code.
+
+![test-app screens](./images/atticandbutton.png)
+
+The demo works as follows:
+* Configuration starts by setting up a Project, Feature, and Event in the Optimizely dashboard, which will be queried by the demo webpage using the Optimizely Python API.
+* The demo consists of a single webpage stored in **./templates/index.html** and a local server listening on Port 4001 that runs the server.
+* The main function and UI event handlers for the webpage are defined in **./application.py**.
+* The code to create an Optimizely Client instance is defined in **./optimizely_config_manager.py**.
+* As the webpage is rendered, the list of products and the names of their associated images are read from **./items.csv**.
+* When the user clicks **Shop**, the demo invokes `activate`, passing the user ID entered in the edit field and the experiment ID (`item_sort`). The code then uses the Variation key returned to sort the items accordingly and passes it to the underlying HTML to be used during rendering.
+* When the user clicks one of the **Buy Now** buttons, the demo tracks the event by invoking `track`, passing in the event key `item_purchase` and the user ID entered in the edit field.
+
+## Prerequisites
+* [Optimizely account](https://app.optimizely.com/signin)
+* [Python 2.7+](https://www.python.org) including pip and virtualenv
+
+## Quick start
+This section shows you how to prepare the project in the Optimizely portal and run the demo in a browser hosted by a local server.
+
+### Preparing an Optimizely Project
+This section provides the steps to prepare an Optimizely project in your dashboard.
+
+1. Clone or download the **python-sdk-demo-app** package.
+2. Log in or create an [Optimizely Account](https://app.optimizely.com/signin).
+3. Create a project via the Optimizely dashboard.
+4. Add a Feature Test with the key `item-sort` and two Variation keys: `price` and `category`. These will act as a toggle for the type of sorting to be permitted.
+5. Add an Event key called `item_purchase` and save the event. This event will be triggered when one of the **Buy** buttons is clicked.
+6. Navigate to the directory where the package was downloaded to in Step 1 and open **./application.py** in a text editor.
+7. Update the following:
+ * `<project_id>`: set to the ID of the project created in Step 3.
```

Comment:
We should refrain from using project ID based URLs and instead resort to ask user to define datafile URL. Thoughts @mikeng13 @ceimaj ?

Reply:
@aliabbasrizvi I haven't revised the docs for this comment since this appears to be an internal eng. conversation.